src/server/scripts/Outland/zone_blades_edge_mountains.cpp

```diff
@@ -33,75 +33,164 @@ EndContentData */
 enum deathsDoorWrapGate
 {
     SPELL_ARTILLERY_ON_THE_WRAP_GATE            = 39221,
-    SPELL_ANTI_DEMON_FLAME_THROWER              = 39222,
+    SPELL_IMP_AURA                              = 39227,
+    SPELL_HOUND_AURA                            = 39275,
+    SPELL_EXPLOSION                             = 30934,
+
+    NPC_EXPLOSION_BUNNY                         = 22502,
+    NPC_FEL_IMP                                 = 22474,
+    NPC_HOUND                                   = 22500,
     NPC_NORTH_GATE                              = 22471,
     NPC_SOUTH_GATE                              = 22472,
     NPC_NORTH_GATE_CREDIT                       = 22503,
     NPC_SOUTH_GATE_CREDIT                       = 22504,
 
+    GO_FIRE                                     = 185317,
+    GO_BIG_FIRE                                 = 185319
 };
 
 class npc_deahts_door_wrap_gate : public CreatureScript
 {
 public:
     npc_deahts_door_wrap_gate() : CreatureScript("npc_deahts_door_wrap_gate") { }
 
-    CreatureAI* GetAI(Creature* creature) const
-    {
-        return new npc_deahts_door_wrap_gateAI(creature);
-    }
-
     struct npc_deahts_door_wrap_gateAI : public ScriptedAI
     {
-        npc_deahts_door_wrap_gateAI(Creature *c) : ScriptedAI(c)
+        npc_deahts_door_wrap_gateAI(Creature* creature) : ScriptedAI(creature) { }
```

Comment:
Travis warning tells you to do this:

npc_deahts_door_wrap_gateAI(Creature* creature) : ScriptedAI(creature), PartyTime(nullptr) { }

Reply:
actually, try o(nullptr)